tests/unittests/unittest.cpp

```diff
@@ -1385,3 +1393,30 @@ TEST_CASE("Relations after cloning with SIO", "[relations][basics]") {
 }
 
 #endif
+
+void runRelationAfterCloneEqualCheck() {
+  auto newColl = ExampleMCCollection();
+  auto coll = ExampleMCCollection();
+  coll.create();
+  coll.create();
+  coll[0].addparents(coll[0]);
+  coll[0].adddaughters(coll[1]);
+  coll[0].adddaughters(coll[0]);
```

Comment:
This is its own daughter? Does this test still work if this is removed? I don't see a reason why it wouldn't but it looks a bit odd, IMHO.

Reply:
It works. Adding the object itself to the relation is also a possible test case maybe different from adding something else